appside/model/dbt_worksheet_model.js

```diff
@@ -376,10 +398,25 @@ class DbtWorksheetModelConfig {
         return this;
     }
 
+    /**
+     * Setter for this.self_report, tells the model whether to offer self report frame.
+     * @param value - boolean to set it to
+     * @return this
+     */
```

Comment:
All the setter names should start with `set_`. An object can't have a property and a method with the same name. We might have fixed some of these together in a different branch.

Reply:
fixed